spec/mail/encodings_spec.rb

```diff
@@ -133,7 +133,7 @@
     it "should decode a long encoded string" do
       string = '=?UTF-8?B?VGhpcyBpcyDjgYIgcmVhbGx5IGxvbmcgc3RyaW5nIFRoaXMgaXMg44GCIHJl?= =?UTF-8?B?YWxseSBsb25nIHN0cmluZyBUaGlzIGlzIOOBgiByZWFsbHkgbG9uZyBzdHJp?= =?UTF-8?B?bmcgVGhpcyBpcyDjgYIgcmVhbGx5IGxvbmcgc3RyaW5nIFRoaXMgaXMg44GC?= =?UTF-8?B?IHJlYWxseSBsb25nIHN0cmluZw==?='
       result = "This is あ really long string This is あ really long string This is あ really long string This is あ really long string This is あ really long string"
-      result.force_encoding('UTF-8') if RUBY_VERSION >= '1.9'
+      result.encode('UTF-8') if RUBY_VERSION >= '1.9'
```

Comment:
Don't you need encode! to change the original string?

[4] pry(main)> temp = 'how are you?'
=> "how are you?"
[5] pry(main)> temp.encode 'ascii-8bit'
=> "how are you?"
[6] pry(main)> temp.encoding
=> #<Encoding:UTF-8>
[7] pry(main)> temp.encode! 'ascii-8bit'
=> "how are you?"
[8] pry(main)> temp.encoding
=> #<Encoding:ASCII-8BIT>
[9] pry(main)>